sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/view/DefaultAggregation.java

```diff
@@ -56,6 +56,13 @@ private static Aggregation resolve(InstrumentDescriptor instrument) {
         .createAggregator(instrumentDescriptor, exemplarFilter);
   }
 
+  @Override
+  public boolean isCompatibleWithInstrument(InstrumentDescriptor instrumentDescriptor) {
+    // This should always return true
```

Comment:
any reason not to just return `true` then?

Reply:
If it returns false, then that means a maintainer of the SDK has configured an incompatible default aggregation for an instrument. It could simply return `true`, but this offers a little more protection and prevents measurements from flowing to the aggregator even in the case of an SDK bug, which hopefully never happens.